src/templates/React InstantSearch widget/package.json.hbs

```diff
@@ -6,6 +6,7 @@
     "instantsearch-widget",
     "instantsearch",
     "react-instantsearch",
+    "react-instantsearch-widget-{{ packageName }}"
```

Comment:
the package name includes the scope, it should probably be kebabCaseName

Reply:
good point https://github.com/algolia/create-instantsearch-app/pull/456/commits/2ea1b177f34bc052e154e2465ede363142aab10e